dirs/dirs.go

```diff
@@ -293,3 +291,27 @@ func SetRootDir(rootdir string) {
 	ErrtrackerDbDir = filepath.Join(rootdir, snappyDir, "errtracker.db")
 	SysfsDir = filepath.Join(rootdir, "/sys")
 }
+
+// what inside a (non-classic) snap is /usr/lib/snapd, outside can come from different places
+func libExecOutside(base string) string {
```

Comment:
The code is ok but the name of the function feels slightly off. The location of libecexdir on the host is a constant. I understand this really means “one of viable libexecdir locations of *snapd*” but the name doesn’t reflect that.

Perhaps snapdCompletionDir or something like that?

Reply:
but we have `CoreLibExecDir = "/usr/lib/snapd"` and `DistroLibExecDir = filepath.Join(rootdir, "/usr/libexec/snapd")`.

Maybe I don't understand your point.

Reply:
I guess the point is that the directory that is CoreLibExecDir inside a snap is not constant, from outside.